chapter_07_aggregate.asciidoc

```diff
@@ -731,26 +749,26 @@ def test_concurrent_updates_to_version_are_not_allowed(postgres_session_factory)
 To get the test to pass as it is, we can set the transaction isolation level
 on our session:
 
-[[transaction_serializable]]
+[[isolation_repeatable_read]]
 .Set isolation level for session (src/allocation/service_layer/unit_of_work.py)
 ====
 [source,python]
 ----
 DEFAULT_SESSION_FACTORY = sessionmaker(bind=create_engine(
     config.get_postgres_uri(),
-    isolation_level="SERIALIZABLE",
+    isolation_level="REPEATABLE READ",
 ))
 ----
 ====
 
 Transaction isolation levels are tricky stuff, it's worth spending time
-understanding https://www.postgresql.org/docs/9.6/transaction-iso.html[the
+understanding https://www.postgresql.org/docs/12/transaction-iso.html[the
 documentation].
```

Comment:
I would make the caveat here about "spending time understanding your vendor's specific interpretation of isolation levels".  A specific example is that Oracle `SERIALIAZABLE` is equivalent to Postgres `REPEATABLE READ`.

Reply:
will do.

one of the nice things in python is that we've decided that the `one --and preferably only one-- obvious way to do it` is Postgres :)